service

```diff
@@ -39,10 +50,6 @@ def main():
     now, ts = current_time()
     last_animation_time_reset = now
     while 1:
-        if now.year < 2000:
-            print >>sys.stderr, "too soon"
-            time.sleep(1)
-            continue
 
```

Comment:
This must stay in as it guards against NTP time not beeing synchronized yet... on the pi the year will be 1970 in that case.

Reply:
done.